mod_wtjshoppingfavorites/src/Fields/ModuleinfoField.php

```diff
@@ -0,0 +1,99 @@
+<?php
+/**
+ * WT JoomShopping Favorites is an alternative wish list (favorite products) for JoomShopping based on cookies.
+ * @package     WT JoomShopping Favorite
+ * @version     2.0.2
+ * @Author      Sergey Tolkachyov, https://web-tolk.ru
+ * @copyright   Copyright (C) 2024 Sergey Tolkachyov
+ * @license     GNU/GPL 3.0
+ * @since       1.0.0
+ * @link        https://web-tolk.ru/en/dev/joomshopping/wt-joomshopping-favorite.html
```

Comment:
Я отключил на сайте суффикс `.html`
```suggestion
 * @link        https://web-tolk.ru/en/dev/joomshopping/wt-joomshopping-favorite
```